pkg/openstack/actuator.go

```diff
@@ -129,6 +136,12 @@ func (a *OpenStackActuator) sync(ctx context.Context, cr *minterv1.CredentialsRe
 	return nil
 }
 
+func (a *OpenStackActuator) prepareCloudsData(clouds string) (string, error) {
+	cloud, err := gophercloudutils.
```

Comment:
Looks like a syntax error? Probably shouldn't commit this, even if it is fixed in the next commit.

Reply:
/hold for this